sdk/configs/examples/bmi270_no_scu/defconfig

```diff
@@ -0,0 +1,9 @@
++CXD56_I2C0_SCUSEQ=n
```

Comment:
Create a `defconfig` using `mkdefconfig.py`.
```
$ cd spresense/sdk
$ ./tools/mkdefconfig.py examples/bmi270_no_scu
```

Reply:
Thanks, teaching usage of tool. I success to generate defconfig